eng/common/TestResources/config-functions.ps1

```diff
@@ -0,0 +1,93 @@
+function ShouldMarkValueAsSecret([string]$serviceDirectoryPrefix, [string]$key, [string]$value, [array]$allowedValues = @())
```

Comment:
Is config too generic here for a file name? Should this be SubConfig-Helpers.ps1 instead?